common/ld2450-base.yaml

```diff
@@ -416,6 +416,47 @@ number:
     initial_value: 15
     disabled_by_default: true
 
+  - platform: template
+    name: "Occupancy Mask 1 Begin X"
+    id: occupancy_mask_1_begin_x
+    max_value: 6000
+    min_value: -6000
+    unit_of_measurement: "mm"
+    step: 10
+    entity_category: config
```

Comment:
I think we should remove the entity_category so that these are grouped in the controls section along with the zone controls